src/main/resources/fabric.mod.json

```diff
@@ -29,7 +29,7 @@
 	],
 
 	"depends": {
-		"minecraft": ">=1.20.2",
-		"fabric-networking-api-v1": ">=3.0.8"
+		"minecraft": ">=1.20.5",
+		"fabric-networking-api-v1": ">=4.0.8"
 	}
 }
```

Comment:
In this file tabs could also be converted to spaces. The mixins one was converted, java code doesn't use tabs, so I guess it's only logical